### ISSUE TITLE
Enable emails to be replied to

### ISSUE THREAD

**Issue by essoen** [capraconsulting/dih-webapp#243]

**Comment by larseen** [comment]
All email sent can be replied to and the recipient is the `frivillig@drapenihavet.no` isnt this the correct behavior @essoen ?

**Comment by essoen** [comment]
Yes, the templates we have now. They also want coordinators to create emails, and these must be able to be replied to.
